docs: update HDI case study

thorbenheins opened the pull request:
<!--   Thank you for your contribution. Before you submit the pull request:
1. Follow our contribution guidelines
2. Test your changes and attach their results to the pull request.
3. Update the relevant documentation.
-->

**Description**

- ...
- ...
- ...

**Related issue(s)**
<!-- If you refer to a particular issue, provide its number, otherwise, remove this section.
For example, `Resolves #123`, `Fixes #43`, or `See also #33`. The `See also #33` option will not automatically close the issue after the PR merge. -->

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Enhanced clarity on HDI Global SE's operational practices for asynchronous communication.
	- Updated revenue format for better readability.
	- Introduced a forward-looking perspective on AsyncAPI document usage.
	- Highlighted public accessibility of the documentation wiki for developers.

- **Documentation**
	- Rephrased challenges and solution descriptions to reflect current practices and methodologies.
	- Clarified the approach to AsyncAPI and GitOps practices.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
reviewed the article, made some changes for consistency

Commit:
i18n numbers

Commit:
Postponing the decision whether AsyncAPI files should be used as our config. Added reference to IDP.

thorbenheins committed:
Merge pull request #1 from thorbenheins/feature/review-changes

reviewed the article, made some changes for consistency

asyncapi-bot committed:
Merge branch 'master' into master

derberg commented on config/casestudies/hdi.yml:
```diff
@@ -198,7 +200,9 @@ asyncapi: #More specific details about AsyncAPI itself and how is it used.
     - The AsyncAPI documents are used for documentation purposes by the platform team. 
       It provides a comprehensive overview of the asynchronous communication in our system, including the available topics and the structure of the messages.
-    - (Future Plans) The AsyncAPI document will be used to generate Java DTOs using the AsyncAPI Java generator, ensuring type safety and clear data structure understanding.
+    - (Outlook) The AsyncAPI document will be used to generate Java DTOs using the AsyncAPI Java generator, ensuring type safety and clear data structure understanding.
```
for simple java class generation, instead of generator, I suggest modelina https://github.com/asyncapi/modelina

AsyncAPI Generator and templates developers rather think of clients, docs, sdk, and stuff like that - for classes, we found out some time ago that better approach is to separate it into separate library.

so best maybe just write that outlook is to generate DTOs, and not much important what will you use for it

thorbenheins replied:
```suggestion
    - (Outlook) The AsyncAPI document will be used to generate Java DTOs, ensuring type safety and clear data structure understanding.
```

derberg committed:
Merge branch 'master' into master

thorbenheins committed:
Update config/casestudies/hdi.yml

removing reference to specific library to use, in order avoid confusion and not send dev running in the wrong direction, as modelina is the correct approach.

sambhavgupta0705 commented:
/update

asyncapi-bot committed:
Merge branch 'master' into master

derberg committed:
Merge branch 'master' into master

derberg commented:
Thanks @thorbenheins !

/rtm
